internal/pkg/crawl/crawl.go

```diff
@@ -163,11 +163,13 @@ func (c *Crawl) Start() (err error) {
 	} else {
 		// Push the seed list to the queue
 		c.Log.Info("Pushing seeds in the local queue..")
+		var seedPointers []*queue.Item
 		for _, item := range c.SeedList {
-			item := item
-			c.Queue.Enqueue(&item)
+			seedPointers = append(seedPointers, &item)
 		}
+		c.Queue.BatchEnqueueUntilCommited(seedPointers...) // TODO: chunk
```

Comment:
On my machine, `BatchEnqueueUntilCommited()` took 10～20 seconds for 10 million items. (resulted in 2.6G queue file)

No one here is using `get list` to load hundreds of millions of URLs, right?
If so, it would be better to call `BatchEnqueueUntilCommited()` once for every few million URLs.
:)

Reply:
done via 1c80c199e6fa98cdb581a15dc470cc6ab4ea602e